catalog/nsili/catalog-nsili-transformer/src/main/java/com/connexta/alliance/nsili/transformer/DAGConverter.java

```diff
@@ -1173,7 +1173,7 @@ public static void logMetacard(Metacard metacard, String id) {
                     String valueStr = getValueString(attribute.getValues());
                     LOGGER.trace("{} :  {} : {}", id, descriptor.getName(), valueStr);
                 } else {
-                    LOGGER.trace("{} : {} : {}", id, descriptor.getName(), attribute.getValue());
+                    LOGGER.trace("{} :  {} : {}", id, descriptor.getName(), attribute.getValue());
```

Comment:
Nit - extra space inserted here.  Surprised checkstyle didn't complain about this.